### ISSUE TITLE
Feature request - Option to show our hide smart feeds

### ISSUE THREAD

**Issue by githubnavigator** [Ranchero-Software/NetNewsWire#2136]
Another idea: Personally, I never use Smart Feeds on any newsreader app I have used before, even though I like the idea. To save screen real estate, maybe consider having the option to not display the group at all (rather than just collapse it). Thanks again everyone!

**Comment by peternlewis** [comment]
I disagree. Yes, the latter would solve my specific problem, but its not a duplicate.

The state of disclosure should be preserved regardless of whether the folders can be hidden.
